XcodeServerSDK/XcodeServer.swift

```diff
@@ -186,23 +186,55 @@ public extension XcodeServer {
             completion(success: false, error: Error.withInfo("Nil response"))
         }
     }
-
-    public func createBot(botOrder: Bot, completion: (bot: Bot?, error: NSError?) -> ()) {
-        
-        let body: NSDictionary = botOrder.dictionarify()
+    
+    public enum CreateBotResponse {
+        case Success(bot: Bot)
+        case BlueprintNeedsFixing(fixedBlueprint: SourceControlBlueprint)
+        case Error(error: ErrorType)
```

Comment:
yeah sorry about that; I added a comment below explaining that I completely missed that this is for asyn calls.

Have a good one, go rest :zzz: